### PR TITLE
Update cert-manager version

### PR DESCRIPTION
**Pull request by btat**
Fixes https://github.com/rancher/rancher-docs/issues/511

### CODE REVIEW

**Comment by martyav** [comment]
Unfortunately, the changes to cert-manager haven't been tested in versions older than 2.7.

**Comment by btat** [comment]
> Unfortunately, the changes to cert-manager haven't been tested in versions older than 2.7.

Test cases included 2.6 based on https://github.com/rancher/rancher/issues/40880#issuecomment-1473636555. But upon further inspection we actually need to incorporate tabs to account for different cert-manager + k8s combinations.

Rancher 2.6 supports K8s 1.20 - 1.24
cert-manager 1.11 supports K8s 1.21 - 1.26
cert-manager 1.7 supports K8s 1.18 - 1.23

File: docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/publish-images.md
Comment (JonCrowther) [inline docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/publish-images.md:1]: There's a reference to helm version at line 243:
```
helm fetch jetstack/cert-manager --version v0.12.0
```

**Review by JonCrowther — COMMENTED** [review]
Missed a couple references to the Helm version. Otherwise this looks good and I will close https://github.com/rancher/rancher-docs/pull/527

File: docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md [hunk 1]
```diff
@@ -74,21 +74,14 @@ When setting up the Rancher Helm template, there are several options in the Helm
...
-
 By default, Rancher generates a CA and uses cert-manager to issue the certificate for access to the Rancher server interface.
 
-:::note
-
-Recent changes to cert-manager require an upgrade. If you are upgrading Rancher and using a version of cert-manager older than v0.11.0, please see our [upgrade cert-manager documentation](../../resources/upgrade-cert-manager.md).
-
-:::
-
 ##### 1. Add the cert-manager repo
```
Comment (martyav) [inline docs/getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md:85]: Nitpick about inconsistent capitalization on existing page headings

**Review by martyav — COMMENTED** [review]
Don't really have anything to add in regards to wording. The existing pages have inconsistent capitalization in the sections headers, which doesn't look great, but isn't a blocker. 

Sorry for the misunderstanding about which versions have been validated -- there was some confusion on Friday and I didn't verify against the table of test cases in https://github.com/rancher/rancher/issues/40880#issuecomment-1473636555.

File: versioned_docs/version-2.6/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md [hunk 2]
```diff
@@ -121,12 +121,6 @@ There are three recommended options for the source of the certificate used for T
 
 ### 4. Install cert-manager
 
-:::note
-
-New in v2.6.4, cert-manager versions 1.6.2 and 1.7.1 are compatible. We recommend v1.7.x because v 1.6.x will reach end-of-life on March 30, 2022.
-
-:::
-
```
Comment (cbron) [inline versioned_docs/version-2.6/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md:129]: I think you should leave this in all places until 2.6 is EOL, which is September 2023.

File: docs/getting-started/installation-and-upgrade/resources/upgrade-cert-manager.md [hunk 3]
```diff
@@ -259,13 +259,6 @@ cert-manager-webhook-787858fcdb-nlzsq      1/1     Running   0          2m
 
 ## Cert-Manager API change and data migration
 
----
-_New in v2.6.4_
-
-Rancher now supports cert-manager versions 1.6.2 and 1.7.1. We recommend v1.7.x because v 1.6.x will reach end-of-life on March 30, 2022. To read more, see the [cert-manager docs](../../../pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md#4-install-cert-manager). For instructions on upgrading cert-manager from version 1.5 to 1.6, see the upstream cert-manager documentation [here](https://cert-manager.io/docs/installation/upgrading/upgrading-1.5-1.6/). For instructions on upgrading cert-manager from version 1.6 to 1.7, see the upstream cert-manager documentation [here](https://cert-manager.io/docs/installation/upgrading/upgrading-1.6-1.7/).
-
----
-
```
Comment (cbron) [inline docs/getting-started/installation-and-upgrade/resources/upgrade-cert-manager.md:268]: Same as above, lots of people still on early 2.6's.

**Review by cbron — COMMENTED** [review]
We should try to merge this before the 2.7.2 release, as users will be checking this version when they plan their upgrades.

File: versioned_docs/version-2.6/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md (hunk 2 above)
Comment (martyav) [reply versioned_docs/version-2.6/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md]: Updated in [4118b81](https://github.com/rancher/rancher-docs/pull/526/commits/4118b811ba8c58e7242be697be3da8ccec1361f9)

File: docs/getting-started/installation-and-upgrade/resources/upgrade-cert-manager.md (hunk 3 above)
Comment (martyav) [reply docs/getting-started/installation-and-upgrade/resources/upgrade-cert-manager.md:268]: I'm going to merge this into the release branch, then create a new working branch to address this problem with the notes. The way I'm currently updating the PR makes it awkward to edit multiple pages at once.

**Pull request merged**